src/components/Menu.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class Menu extends React.Component {
+    constructor() {
+        super()
+        this.state = {
+            menu: [
+                {
+                    value: 'foo'
+                },
+                {
+                    value: 'foo'
+                },
+                {
+                    value: 'foo'
+                },
+                {
+                    value: 'foo'
+                }
+            ]
+        }
+    }
+
+    render() {
+        const { menu } = this.state
+
+        return (
+            <div className="menu">
+                <ul className="menu__list">
+                    {menu.map((item) => (
+                        <li key={item.toString()} className="menu__item">
```

Comment:
You can also use `index` if your items will not change (won't be deleted or re-ordered):

```jsx
{menu.map((item, index) => (
  <li key={index}>...</li>
)}
```